traffic_ops/traffic_ops_golang/federations/federations.go

```diff
@@ -285,17 +258,21 @@ WHERE
 ORDER BY
   ds.xml_id
 `
-	imsQuery := `SELECT max(t) from (
-		SELECT max(fds.last_updated) as t FROM
-  federation_deliveryservice fds
-  JOIN deliveryservice ds ON ds.id = fds.deliveryservice
-  JOIN federation fd ON fd.id = fds.federation
-  JOIN federation_tmuser fu on fu.federation = fd.id
-  JOIN tm_user u on u.id = fu.tm_user
-WHERE
-  u.username = $1
-	UNION ALL
-	select max(last_updated) as t from last_deleted l where l.table_name='federation_deliveryservice') as res`
+	imsQuery := `SELECT max(t) FROM (
+    (SELECT GREATEST(fdsmax, ffrmax, fdmax) AS t FROM
+        (SELECT max(fds.last_updated) AS fdsmax, max(ffr.last_updated) AS ffrmax, max(fd.last_updated) AS fdmax FROM federation_deliveryservice fds
+        JOIN federation_federation_resolver ffr ON ffr.federation = fds.federation
+        JOIN federation fd ON fd.id = fds.federation
+		JOIN federation_tmuser fu on fu.federation = fd.id
+		JOIN tm_user u on u.id = fu.tm_user
+		WHERE u.username = $1
+    ) AS t
+    UNION ALL
+    SELECT max(last_updated) AS t FROM last_deleted l 
+		WHERE l.table_name='federation_deliveryservice' 
```

Comment:
nit: this can be `WHERE l.table_name IN ('federation_deliveryservice', 'federation', 'federation_federation_resolver')`

nit: we should run this through a SQL formatter so it's easier to read (maybe https://www.dpriver.com/pp/sqlformat.htm?)